magma/interface.py

```diff
@@ -52,22 +52,46 @@ class _Interface(Type):
     def __str__(self):
         return str(type(self))
 
+    def make_wire_str(self, driver, value):
```

Comment:
let's make these `_make_wire_str()` and `_make_wires()` unless we expect the outside world to use them